.github/workflows/test-on-centos.yml

```diff
@@ -0,0 +1,42 @@
+name: Run tests in Centos container
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 1 * * *'
+
+jobs:
+  tests-on-centos:
+    strategy:
+      matrix:
+        version: ["c8s", "c9s"]
+    name: "Unittests on Centos ${{ matrix.version }}"
+    runs-on: ubuntu-latest
+    steps:
+      - name: "Checkout"
+        uses: actions/checkout@v4
+      - name: "Run in container"
+        uses: addnab/docker-run-action@v3
+        with:
+          image: quay.io/osbuild/osbuild-ci-${{ matrix.version }}
+          options: --privileged -v ${{ github.workspace }}:/osbuild
+          run: |
+            cd /osbuild
```

Comment:
Is this really needed, since the command to execute tests is using absolute paths? :thinking:

Reply:
Fact, removing. `--rootdir /osbuild` in pytest is doing the trick.